src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs

```diff
@@ -1076,10 +1076,7 @@ private void WriteRedType(TreeType node)
                             WriteLine("get");
                             OpenBlock();
                             WriteLine("var slot = ((Syntax.InternalSyntax.{0})this.Green).{1};", node.Name, CamelCase(field.Name));
-                            WriteLine("return slot != null");
-                            WriteLine("    ? new SyntaxToken(this, slot, {0}, {1})", GetChildPosition(i), GetChildIndex(i));
-                            WriteLine("    : default;");
-                            CloseBlock();
+                            WriteLine($"return slot != null ? new SyntaxToken(this, slot, {GetChildPosition(i)}, {GetChildIndex(i)}) : default;"); CloseBlock();
```

Comment:
Possible that you took care of this in a later commit, but that `CloseBlock();` needs to go on a new line.